src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = React.memo(
+  () => {
+    const [people, setPeople] = useState<Person[]>([]);
+    const [isLoading, setIsLoading] = useState(false);
+    const [isDataLoaded, setIsDataLoaded] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    const loadPeople = async () => {
+      setIsLoading(true);
+
+      try {
+        const loadedPeople = await getPeople();
+
+        const preparedPeople = loadedPeople.map(person => {
+          const mother = loadedPeople.find(mom => (
+            mom.name === person.motherName
+          ));
+          const father = loadedPeople.find(dad => (
+            dad.name === person.fatherName
+          ));
+
+          return (
+            {
+              ...person,
+              mother,
+              father,
+            }
+          );
+        });
+
+        setPeople(preparedPeople);
+        setIsDataLoaded(true);
+      } catch (error) {
+        setErrorMessage('Can\'t load people');
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    useEffect(() => {
+      loadPeople();
+    }, []);
+
+    const isLoadedPeopleExist = isDataLoaded && people.length;
+    const isLoadedPeopleNotExist = isDataLoaded && !people.length;
```

Comment:
Or change variable name

```suggestion
    const isLoadedPeopleExist = isDataLoaded && people.length;
    const isLoadedPeopleNotExist = !isLoadedPeopleExist;
```

Reply:
if I change according to your suggestion it will work incorrectly (see screenshot below)
![image](https://user-images.githubusercontent.com/115103012/215125349-edea1186-b1fc-4233-a6d1-2d831805415b.png)